src/components/manage/History/__snapshots__/History.test.jsx.snap

```diff
@@ -241,7 +241,7 @@ exports[`History renders a history component 1`] = `
           >
             <time
               dateTime="2017-04-19T14:09:34+02:00"
-              title="Wednesday, April 19, 2017 at 12:09 PM"
+              title="4/19/2017, 12:09:34 PM UTC"
```

Comment:
We don't want UTC, we want local time shown.

Reply:
Take into account, that if not noted in the TZ properties of the date string, it's UTC always.